infra/src/main/resources/i18n/translations.fi.json

```diff
@@ -1504,7 +1504,15 @@
     "user-card": {
         "my-details": "Omat tiedot",
         "role": "Rooli",
-        "permissions": "Oikeudet"
+        "permissions": "Oikeudet",
+        "selected-role": "Valittu"
+    },
+    "user-roles": {
+        "operator": "Operaattori",
+        "browser": "Selaaja",
+        "consultant": "Konsultti",
+        "authority": "Virastokäyttäjä",
+        "team": "Kehitystiimi"
```

Comment:
Nämä ovat siis myös kannassa vielä olemassa. Vastaavasti kannasta löytyy käännökset niille permissioneille. Pitäisikö nämä poistaa kannasta+siirtää myös permissionin käännökset tänne? Käytetäänkö kantaan tallennettuja käännöksiä jossain muualla?

Reply:
Itse suosisin niillekin avainpohjaisia käännöksiä. Up to you että haluatko tehdä tässä yhteydessä vai tehdäänkö siitä oma tikettinsä. Permissioneilla nimikentät on rajoitettu 30 merkkiin, enkä itse lähtenyt haastamaan sitä oikkamuutosjutuissa. Käytännössä tuo tekee oikkien nimistä melko lyhyitä (omasta mielestäni vähän turhankin lyhyitä UI:lle tietyissä tapauksissa)

Reply:
Joo ei noita tarvitse oikeastaan kannassa olla, varmaan olisi loogisempaa että ne on täällä. Voi tosiaan olla oma tiksunsa :+1: